api/envoy/config/core/v3/base.proto

```diff
@@ -444,6 +463,27 @@ message RetryPolicy {
   // defaults to 1.
   google.protobuf.UInt32Value num_retries = 2
       [(udpa.annotations.field_migrate).rename = "max_retries"];
+
+  // Specifies the conditions under which retry takes place. These are the same
+  // conditions documented for :ref:`config_http_filters_router_x-envoy-retry-on` and
+  // :ref:`config_http_filters_router_x-envoy-retry-grpc-on`.
```

Comment:
Link all of these back as well. I think instead of duplicating comments and/or saying "This is a copy", I'd just have a comment that said "See :ref:`RetryHostPredicate <envoy_v3_api_field_config.route.v3.RetryPolicy.retry_host_predicate>`.".

That way we have comments in a single place and the appropriate breadcrumbs for the reader to follow to canonoical comments.

Reply:
ahh, that makes sense. 
Done.